### ISSUE TITLE
What order status should be for Medication Order table??

### ISSUE THREAD

**Issue by vaishvikr** [clif-consortium/CLIF#11]
We have the follow order status possible. 
![image](https://github.com/user-attachments/assets/599ce508-3ab8-471f-aae5-6dc4621b04f1)
The documentation does not mention which type of order status should we keep in CLIF Med Order Table.
![image](https://github.com/user-attachments/assets/7c5d4fa4-441a-4b19-a0d6-a3a28c741551)

**Comment by 08wparker** [comment]
sounds like we need `med_order_status_name` and `med_order_status_category`

**Comment by 08wparker** [comment]
@rbaccile is constructing this as well

**Comment by baanna23** [comment]
Agree, med_order_status_name (verified, discontinued, cancelled, etc.) and med_order_status_category (active/cancelled) make sense. In @vaishvikr example would map cancelled and discontinued into cancelled category and verified/completed/dispensed into the active category.

**Comment by rbaccile** [comment]
I put this in slack already, but adding it here:

There is no medication order status in the medication order table at UC, and I am not convinced it truly is a table containing all the medication orders because it has much fewer rows than the med_out or the med_admin tables. @08wparker

**Comment by deetherese** [comment]
Table moved to new alpha maturity level, also added ordering_provider_id https://docs.google.com/presentation/d/1BrnR4IL7S2OnA8AEE08KMfAqQV69l3Y0/edit?slide=id.g3ee8ea16ead_0_37#slide=id.g3ee8ea16ead_0_37
